### DISCUSSION TITLE
Laser Engraver Recommendations

### DISCUSSION THREAD

**Discussion by LDHTHOPI** [arkypita/LaserGRBL#2225] [General]
I have a $300 credit on Amazon to purchase a new laser engraver.  Any suggestions?  I use it to engrave logos onto wood, bamboo, and other materials.  I may also use it to cut stickers and possibly some foam.  Thanks!

**Comment by Tribo-2dot0** [comment]
Two Trees TTS-55 é um ótimo custo benefício...
https://www.twotrees3dofficial.com/products/tts-55-pro-tts-10-pro-diode-laser-engraver-twotrees
